budget app backend/controllers/account_controller.js

```diff
@@ -1,53 +1,90 @@
-const users = require("../models/users");
+const accounts = require("../models/accounts");
 const mongoose = require("mongoose")
 
 module.exports = class {
     static async get_all_accounts(req, res) {
-        const { user_id } = req.body
-        const accounts = await users.findById(user_id)
-
+        const results = await accounts.find({
+            user_id: req.user._id
+        })
         res.status(200).json({
             ok: true,
-            accounts,
-            message: `All accounts GET!`
+            accounts: results,
+            message: `All accounts of user (${req.user._id})`
         })
     }
     static async get_account_by_id(req, res) {
-        res.status(200).json({
-            ok: true,
-            message: `${req.params.id} account GET!`
-        })
+        const result = await accounts.findById(req.params.id)
+        if(result){
+            res.status(200).json({
+                ok: true,
+                account: result,
+                message: `${req.params.id} account details sent.`
```

Comment:
Success responses doesn't need to have message. Remove it, please everywhere. The same with 'ok' field